src/core/server/logging/logging_config.ts

```diff
@@ -59,7 +60,7 @@ export const loggerSchema = schema.object({
 export type LoggerConfigType = TypeOf<typeof loggerSchema>;
 export const config = {
   path: 'logging',
-  schema: schema.object({
+  schema: legacyLoggingConfigSchema.extends({
```

Comment:
Core's `logging` schema now extends the legacy config schema. We will remove that when we totally get rid of the legacy logging system.